src/Compilers/CSharp/Test/Symbol/Symbols/Retargeting/RetargetingTests.cs

```diff
@@ -750,6 +750,250 @@ class C1<T>
             Assert.Equal(c1.MangleName, c1r.MangleName);
             Assert.Equal(c1.MetadataName, c1r.MetadataName);
         }
+
+        [Fact]
+        public void FunctionPointerRetargeting_FullyConsistent()
```

Comment:
>public [](start = 8, length = 6)

```[CompilerTrait(CompilerFeature....)]```? And for other new tests

Reply:
I'll do this in a separate PR. I haven't added a CompilerFeature flag nor have I done this for any existing tests. I'd want to either do everything or nothing.

---
In reply to: [418119854](https://github.com/dotnet/roslyn/pull/43625#discussion_r418119854) [](ancestors = 418119854)